client/__tests__/LandingPageTest/SignInModal.test.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+import { SignInModal } from '../../components/Index/SignInModal.jsx';
+
+const props = {
+  signInRequest: jest.fn(),
+  googleAuthRequest: jest.fn(),
+  forgetPasswordRequest: jest.fn()
+};
+
+describe('<SignInModal />', () => {
+  it('component should render correctly', () => {
+    const wrapper = shallow(<SignInModal { ...props } />);
+    expect(wrapper.node.props.className).toEqual('row modal');
+    expect(wrapper.node.props.id).toEqual('modal2');
+  });
+  describe('Test for onChange event', () => {
```

Comment:
### Nitpick
Description sounds too technical make it more friendly for a non-technical person to understand